src/Core/Services/Impl/ExpenseReportBuilder.cs

```diff
@@ -13,7 +14,7 @@ public ExpenseReportBuilder(INumberGenerator numberGenerator, ICalendar calendar
 			_calendar = calendar;
 		}
 
-		public ExpenseReport Build(Employee creator)
+		public async Task<ExpenseReport> Build(Employee creator)
```

Comment:
If you are only calling synchronous code, I recommend removing the async from `Build()` and returning `Task.FromResult(expenseReport)`.  `Task.FromResult` will create a completed task.  Be careful about synchronously blocking in your task returning `Build()`  method as this would be unexpected for people calling `Build()`